net/port.go

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package net
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+)
+
+// IPFamily refers to a specific family if not empty, i.e. "4" or "6"
+type IPFamily string
+
+// Constants refering to IPv4 and IPv6
+const (
+	IPv4 IPFamily = "4"
```

Comment:
Isn't there already an enum defined for this in this package?  I thought there was..

Reply:
I wasn't able to find one

Reply:
This is the enum defined in the API
https://github.com/kubernetes/api/blob/573bfcf8bde3ccf7d9ea884b33325e3c8322fa20/core/v1/types.go#L3846-L3854

an example of its usage 
https://github.com/kubernetes/kubernetes/blob/5708511499fe500ae3b4bbd40204cef382f652e8/test/e2e/network/dual_stack.go#L219-L221